dev-infrastructure/docs/development-setup.md

```diff
@@ -317,234 +317,130 @@ Then register it with the Maestro Server
-
-# Generate a provision_shards.config for port-forwarded maestro ...
-make -C $the_aro_hcp_dir/cluster-service provision-shard > provision_shards.config
-
-# the resulting configuration requires two portforwardings into the service cluster
-kubectl port-forward svc/maestro 8001:8000 -n maestro
-kubectl port-forward svc/maestro-grpc 8090 -n maestro
-
-# Alternatively, update provision shards config with new shard manually
-cat <<EOF > ./provision_shards.config
-provision_shards:
-- id: 1
-  maestro_config: |
+## Creating an ARO HCP Cluster via Cluster Service
+
+### Creating a cluster
+1) Login to your CS deployment
+  - Access your CS deployment locally
+    ```bash
+    kubectl port-forward svc/clusters-service 8000:8000 -n cluster-service
+    ```
+  - Login to your CS deployment
+    ```bash
+    ocm login --url=http://localhost:8000 --use-auth-code
+    ```
+
+2) Create pre-requisite resources for cluster creation
+
+    Replace `resource-group`, `vnet-name`, `nsg-name` and `subnet-name` with any valid names.
+  
+  - Create a resource group for your ARO HCP cluster
+    ```
+    az group create --name <resource-group> --location "westus3"
+    ```
+  - Create a Virtual Network
+    ```
+    az network vnet create -n <vnet-name> -g <resource-group> --subnet-name <subnet-name>
+    ```
+  - Create a Network security group
+    ```
+    az network nsg create -n <nsg-name> -g <resource-group>
+    ```
+  - Associate the created VNet with the subnet of the created NSG
+    ```
+    az network vnet subnet update -g <resource-group> -n <subnet-name> --vnet-name <vnet-name> --network-security-group <nsg-name>
+    ```
+
+3) Create the cluster
+    > **NOTE** See the [Cluster Service API](https://api.openshift.com/#/default/post_api_clusters_mgmt_v1_clusters) documentation
+    > for further information on the properties within the payload below
+
+    ```bash
+    NAME="<INSERT-NAME-HERE>"
+    SUBSCRIPTION_NAME="ARO Hosted Control Planes (EA Subscription 1)"
+    RESOURCENAME="<INSERT-NAME>"
+    SUBSCRIPTION=$(echo $(az account subscription list | jq '.[] | select(.displayName == $SUBSCRIPTION_NAME)' | jq -r '.subscriptionId'))
+    RESOURCEGROUPNAME="<INSERT-NAME>"
+    TENANTID=$(echo $(cat azure-creds.json | jq -r '.tenantId'))
+    MANAGEDRGNAME="<INSERT-NAME>"
+    SUBNETRESOURCEID="<INSERT-NAME>"
+    NSG="<INSERT-NAME>"
+    cat <<EOF > cluster-test.json
     {
-      "rest_api_config": {
-        "url": "http://localhost:8001"
+      "name": "$NAME",
+      "product": {
+        "id": "aro"
+      },
+      "ccs": {
+        "enabled": true
+      },
+      "region": {
+        "id": "westus3"
+      },
+      "hypershift": {
+        "enabled": true
+      },
+      "multi_az": true,
+      "azure": {
+        "resource_name": "$RESOURCENAME",
```

Comment:
'resource_name', 'resource_group_name' and 'managed_resources_group_name' can be a bit confusing, we have some explanation around this in the CS repo. Should we point there or copy the same info here?

Reply:
We can't point to the CS repo as external users will not have access to it. However, all of this information should be available in our API docs which is available publicly in api.openshift.com, see the note at the start of this section. That should contain details to clarify these properties.